docs/src/pages/demos/buttons/buttons.md

```diff
@@ -133,7 +141,8 @@ or if you want to avoid properties collisions:
 import { Link } from 'react-router-dom'
 import Button from '@material-ui/core/Button';
 
-const MyLink = props => <Link to="/open-collective" {...props} />
+// use `ref` instead of `innerRef` with react-router-dom@^5.0.0
```

Comment:
If I understand it correctly, `react-router-dom@^5.0.0` isn't released yet. They will use `forwardRef` internally. Will we still need to use `forwardRef` in the documentation? Shouldn't we say not to use it?

Reply:
Absolutely right. 
```
<Button component={AnyComponent} />
<Button component={React.forwardRef((props, ref) => <AnyComponent {...props} ref={ref} />)} />
```
is equivalent after all. So with `react-router-dom < 5.0.0` one would need to use `forwardRef` and `innerRef` and in `^5.0.0` they use `forwardRef` internally.